### DISCUSSION TITLE
Nous A1T cannot connect to its WIFI

### DISCUSSION THREAD

**Discussion by kiemlicz** [arendst/Tasmota#21669] [Q&A]
Hello,
I've recently bought the Nous A1T   
I cannot connect to its WIFI, always receive:  
`CTRL-EVENT-ASSOC-REJECT` either with code=17 or on RPI code=16  
(tested on laptop PC - debian - and Android devices)

Assume some wifi config (on either side?) yet no documentation exists that describes it

Any chance to get this working ? I'm also assuming many people should use their android phones to bootstrap device thus I shouldn't be alone

Kind regards

**Comment by HofeBY** [comment]
Did you try a update ?
FritzBox ?
.
I have several of this devices webupdateted from tasmota site and own tasmota build´s.
.
First take a look in your Wifi area about networks with simliar name "tasmota-xyz"
Connect this network, then browser open site 192.168.4.1
configure to your Network.
open shown IP for the rest of config.

**Reply by kiemlicz** [reply]
Thanks

```
Did you try a update ?
```

```
 I cannot connect to its WIFI
```
```
yet no documentation exists that describes it
```

Point me please to doc describing how to fwupgrade A1T without the possibility to connect to `tasmota` wifi (that's boootstrap aka initial config)

**Reply by barbudor** [reply]
You can't upgrade if you can't connect to it's wifi

Times ago, the A1 was the same as the Gosund SP111 and could be dismantled by removing a screw in the earth hole. Still be carefull when opening to avoid damaging the antenna
Once you have access to the board, you serial flash it. Check the instructions at templates.blakadder.com

If you have tried 3 or 4 different phones/computer/OS you can also send it back. They may have a problem
